src/Components/App.tsx

```diff
@@ -19,7 +19,6 @@ const App: FC = () => {
   useEffect(() => {
     fetchMonsters()
     .then((data: any) => {
-      console.log(data.results)
       setMonsters(data.results)
     })
   }, [])
```

Comment:
The JS !! (double bang) operator converts any value into a boolean, this could be used to clean up the logic in monsterViewHandler if you wanted to get into one of the weirder,less used JS operators